aws/alarms/cloudwatch.tf

```diff
@@ -516,3 +516,70 @@ resource "aws_cloudwatch_metric_alarm" "cognito_login_outside_canada_warn" {
 
   alarm_description = "Forms: A sign-in by a forms owner has been detected from outside of Canada."
 }
+
+#
+# Service health: these will trigger if expected metrics thresholds are not met
+#
+
+# Submission lambda: no invocations in a given period during core hours
+resource "aws_cloudwatch_metric_alarm" "healthcheck_lambda_submission_invocations_core_hours" {
+  alarm_name          = "SubmissionLambdaNoInvocationsCoreHours"
+  alarm_description   = "HealthCheck - no `submission` invocations in ${local.lambda_submission_expect_invocation_in_period} minutes."
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = 1
+  threshold           = 1
+  treat_missing_data  = "breaching"
+
+  metric_query {
+    id          = "invocations_core_hours"
+    label       = "Invocations (core hours)"
+    expression  = "IF(((HOUR(invocations)>=9 OR HOUR(invocations)<=6)),invocations,1)" # Before 6am or after 9am (UTC) use metric, otherwise return `1`
```

Comment:
This may prove difficult in Staging since the max alarm evaluation period is 1 day and we won't always see that, especially on weekends.

A fix if the testing confirms this would be to have the expressions be env specific:
1. Prod: expect a submission everyday between these hours
1. Staging: do not expect submissions on the weekend and reduce the hour range

Reply:
Since the alarms are not connected to anything we could just ignore them in Staging as we will know that it is expected.

Reply:
what is the rational of disabling the alarm between 6-9AM ?

Reply:
It was the time period I noticed in the Prod metrics where we no longer had a consistent invocation of the function every 30 minutes.  Ideally we'd have a heartbeat submission that we controlled the frequency of so we could know for certain the period we expect to see submissions within.

Once the API is live and allows for auto download/confirm of form submissions we should be able to create a heartbeat using browser automation in a lambda.

Reply:
thank you!